_pytest/python.py

```diff
@@ -131,6 +131,7 @@ def pytest_namespace():
     raises.Exception = pytest.fail.Exception
     return {
         'raises': raises,
+        'not_raises': NotRaisesContext,
```

Comment:
About `wont_raise`... I'm having some second thoughts that I would like you to consider @RonnyPfannschmidt:

1. `pytest.raises` maps well to `assert raises`, and so does `pytest.not_raises` that also maps well to `assert not raises`.
2. `pytest.wont_raises` is a contraction of `will not`, which is not so nice in general;

Hmm on second thought I think we should ask others for their opinion on this. It is important to nail this down right now than regret it later. I'll send an email to the list! 👍